lib/Extractor/Candidates.cpp

```diff
@@ -354,6 +354,9 @@ Inst *ExprBuilder::build(Value *V) {
       ; // fallthrough to return below
     }
   } else if (auto GEP = dyn_cast<GetElementPtrInst>(V)) {
+
+    return makeArrayRead(V);
```

Comment:
The dead code below (and `buildGEP`) should be removed. When we do support GEP as a native instruction, it seems  that `buildGEP` would look very different.